packages/connectkit/src/components/Pages/Confirmation/index.tsx

```diff
@@ -79,7 +81,7 @@ const Confirmation: React.FC = () => {
         ) : (
           <ModalH1>
             <Link href={txURL} target="_blank" rel="noopener noreferrer">
-              Payment completed in {currency}
+              Payment completed
```

Comment:
imo just delete it entirely since there’s already a header that says “Payment Successful”